components/mpas-ocean/bld/namelist_files/namelist_definition_mpaso.xml

```diff
@@ -2759,7 +2759,7 @@ Default: Defined in namelist_defaults.xml
 
 <entry id="config_salinity_restoring_under_sea_ice" type="logical"
 	category="tracer_forcing_activeTracers" group="tracer_forcing_activeTracers">
-Flag to enable salinity restoring under sea ice.  The default setting is false, where salinity restoring tapers from full restoring in the open ocean (iceFraction=0.0) to zero restoring below full sea ice coverage (iceFraction=1.0); below partial sea ice coverage, restoring is in proportion to iceFraction.  If true, full salinity restoring is used everywhere, regardless of iceFraction value
+Flag to enable salinity restoring under sea ice.  The default setting is true, meaning that full salinity restoring is used everywhere, regardless of iceFraction value, except under ice shelves. If false, the salinity restoring tapers from full restoring in the open ocean (iceFraction=0.0) to zero restoring below full sea ice coverage (iceFraction=1.0); below partial sea ice coverage, restoring is in proportion to iceFraction.
```

Comment:
@alicebarthel - I think this should be false to be consistent with other restoring flags. They get set to true in appropriate configurations elsewhere in the scripts.

Reply:
@jonbob can you explain why this should be false?  This doesn't affect simulations unless `config_use_surface_salinity_monthly_restoring` is set to true.  And I think this is only turned on for G compsets.  Any time salinity restoring is on, we want this flag true.  Do you prefer we have this false too and turn to true when G cases are done with the other flag?

Reply:
@vanroekel -- that is what we have been doing, if only to make it clear in fully-coupled cases that there is no restoring. When the scripts know that we're using datm forcing, they do this:
````
if ($OCN_FORCING eq 'datm_forced_restoring') {
	add_default($nl, 'config_use_activeTracers_surface_restoring', 'val'=>".true.");
	add_default($nl, 'config_use_surface_salinity_monthly_restoring', 'val'=>".true.");
	add_default($nl, 'config_salinity_restoring_constant_piston_velocity', 'val'=>"1.585e-6");
	add_default($nl, 'config_salinity_restoring_max_difference', 'val'=>"100.");
	add_default($nl, 'config_salinity_restoring_under_sea_ice', 'val'=>".false.");
````
(except this PR changes that last line to true). It's not a big deal, but I would have a little concern that having this true by default for all configurations could be confusing.

Reply:
I agree that @jonbob's suggestion is clearer.

Reply:
I see now, makes sense to me and an easy fix.  Thanks for the explanation

Reply:
OK, I've updated the default and think this is good to go